src/presentation/controller/addressLookUp/Controller.ts

```diff
@@ -296,27 +297,44 @@ class AddressLookUpController extends AbstractController {
         const data = this.getAddressData(pageType, address);
 
         // store in api
-        const result = await this.limitedPartnershipService.sendPageData(
-          tokens,
-          ids.transactionId,
-          ids.submissionId,
-          pageType,
-          data
-        );
+        let result;
 
+        if (pageType !== AddressLookUpPageType.confirmGeneralPartnerUsualResidentialAddress){
+          result = await this.limitedPartnershipService.sendPageData(
+            tokens,
+            ids.transactionId,
+            ids.submissionId,
+            pageType,
+            data
+          );
+        };
+        if (pageType === AddressLookUpPageType.confirmGeneralPartnerUsualResidentialAddress) {
```

Comment:
i would use if.. else instead of 2 ifs.
have the first if using === and the else will take care of the !== part